intelij-plugin/src/main/java/tech/pantheon/yanginator/plugin/annotator/YangAnnotator.java

```diff
@@ -160,8 +161,10 @@ public class YangAnnotator implements Annotator {
     public void annotate(@NotNull final PsiElement element, @NotNull final AnnotationHolder holder) {
         if (version.equals("")) {
             PsiElement root = holder.getCurrentAnnotationSession().getFile().getFirstChild();
-            while ((root != null) && !(root instanceof YangModuleStmt)) {
-                root = root.getNextSibling();
+            if (!(root instanceof YangSubmoduleStmt)){
```

Comment:
I would make the check inside while, because if there is anything above submodule statement (e.g. comment) the yang-version won't be updated.

Reply:
done